src/components/data-management/AnalysisModal.jsx

```diff
@@ -29,12 +29,16 @@ const NewExperimentModal = (props) => {
 
   const [experimentsList, setExperimentsList] = useState([]);
   const [numFieldsEditing, setNumFieldsEditing] = useState(0);
+  const [isWorking, setIsWorking] = useState(false);
```

Comment:
I don't understand what this `isWorking` state is for. It's `true` when the modal is not visible, and it disables the launch experiment button, which is not visible as the modal is not visible.

Reply:
Yes, I am not sure either, it seems very distantly related to the rest ticket

Reply:
My guess is it is some way to display some kind of loading in the ui between launching an analysis and the ui switching to data processing

Reply:
👋 The goal of `isWorking` is to disable the launch buttons after it has been clicked; under some circumstances, the user has no visual indication that his click had effect. When the dialog becomes visible, we make sure that `isWorking` is reset.

Reply:
Thanks!!